scripts/deployment/maticOracle.json

```diff
@@ -0,0 +1,67 @@
+{
+    "oracles" : [
+        {
+            "targetPairName":"DAI/MATIC",
+            "tokenPairName":"DAI/ETH",
+            "tokenPriceOracle":"0xFC539A559e170f848323e19dfD66007520510085",
+            "maticPairName":"MATIC/ETH",
+            "maticPriceOracle":"0x327e23A4855b6F663a28c5161541d69Af8973302"
```

Comment:
maticPriceOracle address is same for MATIC, USDC, USDT, WBTC. Is this correct?

Reply:
yes, 0x327e23A4855b6F663a28c5161541d69Af8973302 address is common in all our custom Oracles. As we take MATIC/ETH price and using it convert token prices in terms of MATIC.

Reply:
Cool